.github/workflows/full_documentation.yml

```diff
@@ -89,3 +90,56 @@ jobs:
           cname: ${{ env.DOCUMENTATION_CNAME }}
           token: ${{ secrets.GITHUB_TOKEN }}
           python-version: ${{ matrix.python-version }}
+
+  
+  doc-index-stable:
```

Comment:
Let us parametrize this job as two different API references are required. Also, make migrate this to use ansys/actions.

Reply:
That would be great if we can reuse generic actions.